src/metabase/warehouses/api.clj

```diff
@@ -850,6 +851,8 @@
                                         :is_on_demand is_on_demand
                                         :cache_ttl    cache_ttl
                                         :creator_id   api/*current-user-id*}
+                                       (when provider_name
```

Comment:
What's the point of this being merged in when not null? Won't the underlying column have a null value in either case?

Reply:
Right, no need to conditionally add this field - changed https://github.com/metabase/metabase/pull/62057/commits/4bb5b23a1fcbdaeb2172106a89fdf062861c74b5